# Use mathlib and threads in the LDFALGS
# January 24th 2019 by B. Hajgato (Free University Brussels - VUB)
--- configure.orig	2018-06-06 23:28:35.000000000 +0200
+++ configure	2019-01-24 14:27:36.985828861 +0100
@@ -1510,7 +1510,7 @@
             SHLIB_LD_FLAGS="-Wl,-soname,\$(notdir \$@)"
 	    SHLIB_SUFFIX=".so"
 	    SHLIB_LD="${CC} -shared"
-            LDFLAGS="-Wl,--export-dynamic"
+            LDFLAGS="-pthread -lm -Wl,--export-dynamic"
             LD_SEARCH_FLAGS='-Wl,-rpath-link,${LIB_RUNTIME_DIR}'
             LD_LIBRARY_PATH_VAR="LD_LIBRARY_PATH"
             ;;
